WordPress/Classes/ViewRelated/NUX/FancyAlertViewController+LoginError.swift

```diff
@@ -0,0 +1,43 @@
+import UIKit
+
+extension FancyAlertViewController {
+    private enum Constants {
+        static let successAnimationDuration: TimeInterval = 0.8
+    }
+
+    static func siteAddessHelpController() -> FancyAlertViewController {
+        struct Strings {
+            static let titleText = NSLocalizedString("What's my site address?", comment: "Title of alert helping users understand their site address")
+            static let bodyText = NSLocalizedString("Your site address appears in the bar at the the top of the screen when you visit your site in Safari.", comment: "Body text of alert helping users understand their site address")
```

Comment:
Should we say 'your web browser' instead of 'Safari'? I'm not sure which people are more likely to be familiar with. I may be overthinking it :)